import-service/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "import-service",
+  "version": "1.0.0",
+  "description": "Serverless webpack example using Typescript",
+  "scripts": {
+    "test": "jest",
+    "deploy": "sls deploy"
+  },
+  "dependencies": {
+    "aws-sdk": "^2.792.0",
```

Comment:
I suggest to move this package into `devDependencies` to reduce the bundled lambda size.

Reply:
> I suggest to move this package into `devDependencies` to reduce the bundled lambda size.

thanks